nested_open_struct.gemspec

```diff
@@ -6,7 +6,10 @@ Gem::Specification.new do |s|
   s.homepage    = "http://github.com/porras/nested_open_struct"
   s.summary     = "Thin wrapper around OpenStruct to make structs 'nested'"
   s.description = "Thin wrapper around OpenStruct to make structs 'nested'"
- 
+
   s.files        = Dir.glob("lib/**/*") + %w(LICENSE README.md)
   s.require_path = 'lib'
-end
\ No newline at end of file
+
+  s.add_development_dependency 'minitest', '~> 5.8'
+  s.add_development_dependency 'rake',     '~> 10.4'
```

Comment:
Thanks Piotr for the PR and Tobi for the additional input. Although I usually do as Tobi explained (having runtime dependencies in the gemspec and development dependencies in Gemfile), for this minimal gem I would be inclined to buy Piotr's arguments to remove the dependency on bundler (which I introduced just yesterday, only to have minitest). But I'd have a question, maybe one of you two know the answer. With this PR, there's still a Gemfile, a Gemfile.lock and hence a dependency on bundler. Would there be a way of getting rid of it (installing the development dependencies from the gemspec)?

Thank you!